.github/workflows/release.yml

```diff
@@ -8,31 +8,29 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+
     steps:
       - name: Checkout sources
         uses: actions/checkout@v2
+
       - name: Set up JDK 17
         uses: actions/setup-java@v2
+
         with:
           distribution: 'temurin'
           java-version: 17
-      - name: Cache Gradle packages
-        uses: actions/cache@v2
-        with:
-          path: |
-            ~/.gradle/caches
-            ~/.gradle/loom-cache
-            ~/.gradle/wrapper
-            ~/.m2/repository
-          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
-          restore-keys: ${{ runner.os }}-gradle
 
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
-      - name: Release Artifacts
-        run: ./gradlew build --stacktrace --parallel
+
+      - name: Build Artifacts
+        uses: gradle/gradle-build-action@v2
+        with:
+          arguments: build --stacktrace
```

Comment:
Why has parallel been dropped from this?

Reply:
It's already in the gradle.properties

Reply:
Ok.